MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/TextureRender.java

```diff
@@ -377,7 +383,7 @@ private void storeAsGLMatrix(HomographyMatrix homography) {
      */
     public void drawFrame(List<HomographyMatrix> homographyList) {
         Log.d(TAG, "Drawing frame");
-        Log.d(TAG, "Homography list size: " + homographyList.size());
+
```

Comment:
Do you want to leave the `Log.d` message above? Seems like it will spam logcat.

Reply:
I think this was not addressed?

Reply:
Went through and deleted all the unnecessary Log.d(...) messages. All the files should be cleaned up now in terms of logging.